getting-started/try-out-sandbox.md

```diff
@@ -1,17 +1,28 @@
+---
+## Requirements
+* Provide Webhook URL, if file transfer notification is required.
 
-* Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-* Donec tincidunt risus auctor, gravida orci sit amet, porta nunc.
-* Aenean sit amet ipsum semper, pulvinar ipsum at, eleifend felis.
-* In nec sapien vestibulum, laoreet erat eget, vulputate metus.
+---
 
-Ut id aliquet velit. Curabitur eu placerat quam. Maecenas sed tortor erat. Integer at ex tortor. Maecenas rutrum, magna at aliquam interdum, eros mi tincidunt nulla, ac consectetur velit massa id ante. Donec tempus nisi non tempor suscipit. Aliquam convallis mollis lectus a consectetur. Donec lorem lorem, consequat id sapien sit amet, rhoncus sodales erat. Suspendisse semper dolor velit, at ornare velit placerat id. Cras efficitur est sit amet aliquam tincidunt. Cras convallis diam vitae magna pretium efficitur. Integer magna urna, bibendum sed iaculis eget, semper et eros. Aenean viverra finibus ante, eu pellentesque augue viverra ac. Fusce eu tristique nisi. Curabitur dignissim ultricies nibh, dignissim lobortis justo pretium a. Aliquam consequat feugiat tellus in mollis.
 
-Aenean urna justo, aliquam id elit et, pellentesque cursus leo. Maecenas sed tortor nec odio dictum maximus. Cras suscipit gravida nulla et posuere. Nullam lobortis ante efficitur turpis pellentesque rutrum. Etiam ac condimentum lorem. Nulla ligula dolor, efficitur in maximus ut, finibus ac lorem. Duis vel sem ultricies, bibendum nibh at, tempor lectus. Phasellus ac sem sed dui dignissim varius. Nam sed ex id elit cursus mattis.
 
-Phasellus nec magna non odio venenatis tincidunt. Pellentesque at sem lectus. Nullam vel risus eu nibh vestibulum sollicitudin. Aenean nec porta felis. Quisque vitae posuere lorem. Suspendisse iaculis vulputate ornare. Nulla finibus feugiat neque a varius. Sed viverra pellentesque enim in interdum. Donec eget posuere lorem, sed consequat mi. Sed vel faucibus ipsum.
 
-Nunc finibus non nisl in cursus. Phasellus laoreet tincidunt feugiat. Maecenas in est sit amet leo porttitor efficitur vitae ac neque. Mauris convallis augue quis faucibus pharetra. Fusce consectetur elit at metus maximus blandit. Duis pharetra diam velit. Vivamus feugiat laoreet efficitur. Nullam dapibus nec arcu sed ullamcorper. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vivamus aliquam quam diam, id vulputate libero interdum ac. Quisque luctus porttitor urna ut facilisis. Integer vehicula, nisi eu porttitor gravida, est mauris mattis nisl, at consectetur elit tellus ac erat. Praesent libero eros, scelerisque eu diam quis, accumsan maximus arcu. Donec laoreet enim scelerisque congue posuere.
+![Display CFT](./images/CFT.png)
 
+---
+## How to sign up for CFT sandbox access
+
+Onboarding to CFT sandbox is a simple process.
+1. Fill up the [onboarding form.](https://form.gov.sg/#!/60a4cca76179d60012cdacac/preview)
+2. Once request is processed, receive the API key by email.
```

Comment:
2. Once we received the request, we will email the sandbox credentials to the email address provided in the form.